test/Test_CarbonVote.js

```diff
@@ -1,46 +1,61 @@
+// accounts setting in this test environment:
+// by the nature of DSAuth and DSGuard, accounts[0] has root power. 
+// account[1] is master account, it can call writeAvailableVote
+// account[2] can register
+// account[3] can voteFor
+// account[4] can sendGas
+// account [5] can only call non-restricted functions.
+
+import EVMRevert from 'openzeppelin-solidity/test/helpers/EVMRevert';
+
 const CarbonVoteX = artifacts.require("./CarbonVoteX");
 const TokenAddress = "0x337cDDa6D41A327c5ad456166CCB781a9722AFf9";
 const keccak256 = require('js-sha3').keccak256;
+var BigNumber = web3.BigNumber;
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
 
 contract('Test_CarbonVote', async (accounts) => {
-	it("Test register and getPoll", async () => {
-		const carbonVoteXInstance = await CarbonVoteX.deployed();
+	let carbonVoteXInstance;
+	const existPollId = "0x0000000000000000000000000000000000000000000000000000000000000001";
```

Comment:
const EXISTNG_POLL_ID = "0x01";   

Capitalize const name and keep value short

Reply:
it was on purpose.
if we set EXISTNG_POLL_ID = "0x01"
it would be passed to the contract as "0x0100000000000000000000000000000000000000000000000000000000000000"
the assertion would fail as "0x01" != "0x0100000000000000000000000000000000000000000000000000000000000000"